libs/core/src/components/pds-table/pds-table-head-cell/stories/pds-table-head-cell.stories.js

```diff
@@ -0,0 +1,24 @@
+import { html } from 'lit-html';
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  argTypes: extractArgTypes('pds-table-head-cell'),
+  component: 'pds-table-head-cell',
+  decorators: [withActions],
+  title: 'components/Table/Head',
+};
+
+const BaseTemplate = (args) => html`
+<pds-table>
+  <pds-table-head>
```

Comment:
no use of pds-table-body. Do we want to be consistent or look into making sure the schema requires the use pds-table-body?

Reply:
This is a story for the table head, not sure why I need to include the body?

Reply:
You're fine. I didn't look at the complete code, my apologies.